apps/cra-template-mockbee-social-media/template/src/server.js

```diff
@@ -87,7 +87,7 @@ export function makeServer({ environment = "development" } = {}) {
         "/comments/edit/:postId/:commentId",
         editPostCommentHandler.bind(this)
       );
-      this.post(
+      this.delete(
```

Comment:
This change is good. But the documentation will be needed to be updated too. Else the PR looks good to go

Reply:
Ok will do that

Reply:
Is that something I can do or yoj will?

Reply:
Hey @Hamza-777 , you can make that change!

Reply:
@NadaaFarook The docs for this change are correct already.

Reply:
On it.

Reply:
@sohamsshah Made the required changes in the docs.